packages/backend-storage/src/storage_outputs_aspect.ts

```diff
@@ -104,6 +103,7 @@ export class StorageOutputsAspect implements IAspect {
           name: node.name,
           bucketName: node.resources.bucket.bucketName,
           storageRegion: Stack.of(node).region,
+          paths: node.accessDefinition,
```

Comment:
We didn't need to make changes here https://github.com/aws-amplify/amplify-backend/blob/c99636f26e1eac755da5c902f49640222da3b681/packages/backend-output-schemas/src/storage/v1.ts#L3-L7?

We might also be missing some automated testing if we didn't catch this failure, or Zod is not strict here?

Reply:
This is needed to add `paths` to the `amplify_outputs.json` file. `paths` is optional so storage with no access rules defined will not fail and not have `paths` in their `amplify_outputs.json`.

Reply:
Shouldn't the paths be added as optional since we are adding that in the bucketSchema object before stringifying.

Reply:
Got it, and yup I believe Zod is not strict enough here, I'll add tests for both cases of storage with access and without.